components/application-mgt/org.wso2.carbon.device.application.mgt.common/src/main/java/org/wso2/carbon/device/application/mgt/common/services/CategoryManager.java

```diff
@@ -22,16 +22,18 @@
 import org.wso2.carbon.device.application.mgt.common.Filter;
 import org.wso2.carbon.device.application.mgt.common.exception.ApplicationManagementException;
 
+import java.util.List;
+
 /**
  * CategoryManager is responsible for handling add, delete, update opertaions related with {@link Category}
  */
 public interface CategoryManager {
 
     public Category createCategory(Category application) throws ApplicationManagementException;
 
-    public Category editCategory(int applicationId, Category category) throws ApplicationManagementException;
+    public List<Category> getCategories() throws ApplicationManagementException;
```

Comment:
Can you add the method comments as it's an interface? Additionally, you can remove the public, as it's always public in interface.